lib/v2/nju/radar.js

```diff
@@ -17,6 +17,14 @@ module.exports = {
                 target: '/nju/dafls',
             },
         ],
+        exchangesys: [
```

Comment:
```suggestion
        elite: [
```
should be `elite` according to `type_dict`